package.json

```diff
@@ -14,9 +14,10 @@
     "format": "prettier --write .",
     "format:check": "prettier --cache --check .",
     "preinstall": "npx only-allow pnpm",
-    "lint": "FORCE_COLOR=1 turbo lint",
+    "lint": "node ./scripts/lint.mjs",
     "lint:attw": "FORCE_COLOR=1 turbo lint:attw",
     "lint:fix": "FORCE_COLOR=1 turbo lint -- --fix",
+    "lint:inspect": "pnpx @eslint/config-inspector@latest",
```

Comment:
This is a great way to find out what rules apply to specific files

Reply:
til!